skema/img2mml/api.py

```diff
@@ -42,7 +42,11 @@ def get_mathml_from_latex(eqn) -> str:
     """Read a LaTeX equation string and convert it to presentation MathML"""
 
     # Define the webservice address from the MathJAX service
-    webservice = "http://localhost:" + str(PORT)
+    host = os.environ.get('SKEMA_HOST', '127.0.0.1')
```

Comment:
Let's call this SKEMA_MATHJAX_HOST.

Reply:
I went ahead and made this change when addressing a conflict encountered while updating the branch with the latest changes to `main`.

Reply:
Done